api/api-doc/storage_service.json

```diff
@@ -765,6 +765,38 @@
             }
          ]
       },
+      {
+         "path":"/storage_service/keyspace_offstrategy_compaction/{keyspace}",
+         "operations":[
+            {
+               "method":"POST",
+               "summary":"Perform offstrategy compaction, if needed, in a single keyspace",
+               "type":"long",
+               "nickname":"perform_keyspace_offstrategy_compaction",
+               "produces":[
+                  "application/json"
+               ],
+               "parameters":[
+                  {
+                     "name":"keyspace",
+                     "description":"The keyspace to operate on",
+                     "required":true,
+                     "allowMultiple":false,
+                     "type":"string",
+                     "paramType":"path"
+                  },
+                  {
+                     "name":"cf",
+                     "description":"Comma-seperated table names",
+                     "required":false,
+                     "allowMultiple":false,
```

Comment:
Can we use allowMultiple=true instead of a custom collection interface ("Comma-separated")?

I expect the answer is that everything else does it this way, but I have to ask. It's annoying to have to commit another mistake because everything else has the same mistake, but that's life.

Reply:
`seastar::json::json_base_element` doesn't support that yet.

Reply:
Oh well.

Reply:
left the same in v3